python/ray/tests/test_basic.py

```diff
@@ -2649,8 +2649,6 @@ def f():
     output_lines = captured["out"]
     for i in range(200):
         assert str(i) in output_lines
-    error_lines = captured["err"]
```

Comment:
If we remove this check, then we should include a comment that explains what goes wrong if we do check it. Since people (myself included) will be very tempted to bring back this check.